flycheck.el

```diff
@@ -4326,7 +4381,8 @@ non-nil."
   (flycheck-cancel-error-display-error-at-point-timer)
   (when (flycheck-overlays-at (point))
     (setq flycheck-display-error-at-point-timer
-          (run-at-time flycheck-display-errors-delay nil 'flycheck-display-error-at-point))))
+          (run-at-time flycheck-display-errors-delay nil
+                       'flycheck-automatically-display-error-at-point))))
```

Comment:
Same thing here, we can leave this unchanged, since the hooks shouldn't be run when the `defcustom` is nil.